cardano-wallet.cabal

```diff
@@ -0,0 +1,151 @@
+name:                cardano-wallet
+version:             2.0.0
+synopsis:            The Wallet Backend for a Cardano node.
+description:         Please see README.md
+homepage:            https://github.com/input-output-hk/cardano-wallet
+author:              IOHK Engineering Team
+maintainer:          operations@iohk.io
+copyright:           2019 IOHK
+license:             MIT
+license-file:        LICENSE
+category:            Web
+build-type:          Simple
+extra-source-files:  README.md
+cabal-version:       >=1.10
+
+library
+  default-language:
+      Haskell2010
+  default-extensions:
```

Comment:
I noticed the following proposal in our coding standards document:

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#proposal-language-extensions-are-specified-on-top-of-each-module

I suppose that if the proposal is agreed, then we'd eventually delete this section. (Having said that, I don't feel strongly either way.)